Doc/c-api/contextvars.rst

```diff
@@ -107,9 +107,9 @@ Context variable functions:
 .. c:function:: PyObject *PyContextVar_New(const char *name, PyObject *def)
 
    Create a new ``ContextVar`` object.  The *name* parameter is used
-   for introspection and debug purposes.  The *def* parameter may optionally
-   specify the default value for the context variable.  If an error has
-   occurred, this function returns ``NULL``.
+   for introspection and debug purposes.  The *def* parameter specifies
+   a default value for the context variable, or ``NULL`` for no default.
```

Comment:
>  or ``NULL`` for no default.

Is this accurate? I don't see this in code.

Reply:
See? The documentation didn't give that away. :)

Note the little `Py_XINCREF()` here:

https://github.com/python/cpython/blob/bcb094b41f7fe4dd1686c50891d85632fcf0d481/Python/context.c#L829-L830

Reply:
oh yeah! That was hard to recognize.